app/javascript/components/EventsPage/Event.js

```diff
@@ -120,69 +160,175 @@ const Event = ({
     setShowRsvps(!showRsvps);
   };
 
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  }
+
+  const handleChange = (e) => {
+    setEvent(Object.assign({}, event, { [e.target.name]: e.target.value }));
+  };
+
+  const handleCalendarChange = (newValue) => {
+    setValue(newValue);
+
+    setEvent(
+      Object.assign(event, event, { ["event_date"]: newValue.toISOString() })
+    );
+  };
+
```

Comment:
Events can now be edited, Edited events display correctly the app and in the database